client/src/components/Form/Elements/parameters.js

```diff
@@ -128,46 +126,6 @@ export default Backbone.View.extend({
         });
     },
 
-    /** Text input field */
```

Comment:
I would suggest to just leave this legacy code in since the `_fieldText` is used in other context. We can remove it once the select field PR is complete. This makes this PR also more specific, since we just need to make sure to properly cover for actual `text` and `password` fields.